lib.js

```diff
@@ -39,3 +47,42 @@ export function unwarnify () {
     return originalEmit.apply(process, arguments)
   }
 }
+
+/**
+ * Get a new API client configured from env vars.
+ */
+export function getClient () {
+  const store = new StoreConf({ profile: process.env.W3_STORE_NAME ?? 'w3cli' })
+
+  let serviceConf
+  if (
+    process.env.W3_ACCESS_SERVICE_DID &&
+    process.env.W3_ACCESS_SERVICE_URL &&
+    process.env.W3_UPLOAD_SERVICE_DID &&
+    process.env.W3_UPLOAD_SERVICE_URL
```

Comment:
I'm hoping that with did-web we can simplify this down to just one string per service